index.html

```diff
@@ -222,13 +222,13 @@
           contains a machine-readable listing of <a>URI-M</a>s associated to a given <a>URI-R</a>.
         </dd>
       </dl>
-  </section>
+    </section>
     <section id="resource-management">
       <h2>Resource Management</h2>
 
       <section id="general">
         <h2>General</h2>
-	<section id="ldpnr-ixn-model">
+	        <section id="ldpnr-ixn-model">
```

Comment:
Indentation looks off here.

Reply:
Rats, was fixing bad indent and managed to put a tab in...